src/pages/index.js

```diff
@@ -257,361 +291,348 @@ const Page = () => {
+              <BannerWideLink to="/pg_extensions/pg_tuner">
                 EDB Postgres Tuner
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/query_advisor">
+              <BannerWideLink to="/pg_extensions/query_advisor">
                 EDB Query Advisor
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/wait_states">
+              <BannerWideLink to="/pg_extensions/wait_states">
                 EDB Wait States
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/pg_squeeze">
+              <BannerWideLink to="/pg_extensions/pg_squeeze">
                 PG Squeeze
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/wal2json">
+              <BannerWideLink to="/pg_extensions/wal2json">
                 wal2json
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/system_stats">
+              <BannerWideLink to="/pg_extensions/system_stats">
                 system_stats
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/edb_job_scheduler">
+              <BannerWideLink to="/pg_extensions/edb_job_scheduler">
                 EDB Job Scheduler
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/pg_failover_slots">
+              <BannerWideLink to="/pg_extensions/pg_failover_slots">
                 PG Failover Slots
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/pg_extensions/spl_check/">
+              <BannerWideLink to="/pg_extensions/spl_check/">
                 EDB SPL Check
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/tools/edb_sqlpatch">
+              <BannerDivider headingText="Tools" />
+
+              <BannerWideLink to="/tools/edb_sqlpatch">
                 EDB SQL Patch
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/tools/alteruser_utility">
+              <BannerWideLink to="/tools/alteruser_utility">
                 alteruser
-              </IndexCardLink>
+              </BannerWideLink>
 
-              <IndexCardLink to="/language_pack/latest/">
+              <BannerWideLink to="/language_pack/latest/">
                 Language Pack
-              </IndexCardLink>
-            </IndexCard>
+              </BannerWideLink>
+            </BannerWideSubCard>
 
-            <IndexCard
-              iconName={iconNames.HIGH_AVAILABILITY}
-              headingText="High Availability"
+            <BannerWideSubCard
+              iconName={iconNames.INTEGRATION}
+              headingText="Integration"
```

Comment:
Other banner headings of the same level have a link and are clickable. This one is clickable but does not take you anywhere. Should we add a linkk for consistency? Or remove the clickability of the button to avoid confusion.